iceoryx_hoofs/include/iceoryx_hoofs/internal/units/duration.inl

```diff
@@ -461,53 +466,99 @@ inline constexpr Duration Duration::operator*(const T& rhs) const noexcept
 {
     static_assert(std::is_arithmetic<T>::value, "non arithmetic types are not supported for multiplication");
 
-    if (rhs <= static_cast<T>(0) || *this == Duration::zero())
+    if ((rhs <= static_cast<T>(0)) || (*this == Duration::zero()))
     {
         return Duration::zero();
     }
 
     return multiplyWith<T>(rhs);
 }
 
-
+// AXIVION Next Construct AutosarC++19_03-A8.4.7 : Each argument is larger than two words
 template <typename T>
 inline constexpr Duration operator*(const T& lhs, const Duration& rhs) noexcept
 {
     return rhs * lhs;
 }
 
+
+// AXIVION Next Line AutosarC++19_03-A8.4.7 : Each argument is larger than two words
```

Comment:
This rule apparently does also cause more harm than good or does not work properly ...

Nothing to do. Resolve.

Reply:
@MatthiasKillat Yeah, I agree that it doesn't seem to work properly.